dali/base/dadfs.cpp

```diff
@@ -586,6 +587,56 @@ class CFileSuperOwnerLock : protected CFileLockCompound
     {
         return CFileLockCompound::detach();
     }
+    bool initWithFileLock(const CDfsLogicalFileName &logicalName, IRemoteConnection *conn, unsigned timeout, const char *msg, CFileLock &fcl, unsigned fclmode)
```

Comment:
minor: conn is unused and don't think they'll be a future case where it will, so should remove